config/default/manager_auth_proxy_patch.yaml

```
@@ -22,3 +22,5 @@ spec:
       - name: manager
         args:
         - "--metrics-addr=127.0.0.1:8080"
+        - -v=5
```

SGTM. 
Found a nit.
```suggestion
        - --v=5
```